site/en/docs/android/custom-tabs/translate-language/index.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: "layouts/doc-post.njk"
+title: Translate your content
+seoTitle: "Guide: Localize Content in a Custom Tab"
+date: 2023-04-21
+description: How to translate content in a Custom Tab.
+authors:
+  - alexmkatz
+---
+
+When a user opens a Custom Tab, the content in that tab can be translated for their locale. This guide
+explains how to implement automatic translation of Custom Tab content.
+
+## Implementation
+
+Use the following code to add translation to your Custom Tabs intent builder:
+
+```
```

Comment:
Should we instead tell the reader how to get the appropriate locale from the user's device?

Reply:
That probably makes more sense, I would've assumed though that chrome does this automatically. Can you check with eng?

Reply:
I don't think automatic translation is supported but rather just the language set and then the user still needs to manually trigger the translation in order for it to happen. I would remove the automatic bit out of it.

Reply:
In other words, the use case is indeed one where the language needs to be set in the builder, and not pulled from the device?

Reply:
Yes, I think an example similar to the one Sebastian provided in the first comment would be the way to go. Probably it would make sense to make it clear that this ONLY needs to be set if the developer has a use case where they would want to set the translation language in the CCT to a specific one that is not the device's current language (as that one would be picked by default by the translation logic).

Reply:
Like this? (last commit)

Reply:
The userLocale variable needs to be the one passed to the setter (e.g. .setTranslateLocale(userLocale))

Reply:
Done.

Reply:
Current version LGTM!